libs/blocks/section-metadata/section-metadata.css

```diff
@@ -141,6 +141,15 @@
   gap: var(--spacing-xxxl);
 }
 
+.section.container-mobile, 
```

Comment:
At first sight, this addition of `.section.container-mobile` looks like it can cause some issues. For example, the top and bottom padding of the section is now set at `6px` for viewports above `601px`. Is that requested anywhere explicitly? What's the intention behind adding this selector?